packages/core/src/examples/healthchecks.ts

```diff
@@ -0,0 +1,56 @@
+import express from 'express';
+import { startServer } from '../lib/server';
+import { defaultHealthCheck } from '../config/defaults';
+
+const app = express();
+
+console.log('Server with a basic enabled health check');
+startServer(app, {
+  port: 4999,
+  healthCheck: {
+    enabled: true,
+  },
+});
+// {"status":"ok"}
+
+console.log('Server with a disabled health check');
+startServer(app, {
+  port: 4998,
+  healthCheck: {
+    enabled: false,
+  },
+});
+// 404
+
+console.log('Server with a custom enabled health check with a custom path `healthz` with no details');
+startServer(app, {
+  port: 4997,
+  terminusOptions: {
```

Comment:
What happens if you combine both?

```
  healthCheck: {
    enabled: true,
  },
   healthChecks: defaultHealthCheck(false, {
      '/healthz': () => Promise.resolve(),
    }),
```

Reply:
terminus take precedence

Reply:
When enabled, we use both of them, the `/healthcheck` and the `healthz` passed via the terminus options unless you provide the same key `/healthcheck` which would overwrite the one and would behave as if we ve set it to `enabled:false`

`enabled:true` is merely a convenience to not define the end point.

Is this something we d like to keep for convenience, or just remove it and pass it via terminus

Reply:
once healthcehck is pased via enabled, it cannot be updated.
if you d like to pass it via `terminus`, then set it to disabled and have full control.